conans/test/functional/generators/components/test_components_cmake_generators.py

```diff
@@ -57,16 +57,10 @@ def package_info(self):
                     self.cpp_info.filenames["cmake_find_package"] = "MYG"
                     self.cpp_info.set_property("cmake_file_name", "MYG")
 
-                    self.cpp_info.names["cmake_find_package_multi"] = "MyGreetings"
```

Comment:
Just to make sure, the ``.names`` approach doesn't dissapear for 1.X and ``cmake_find_package[multi]`` right? Then maybe it is better to leave the test as it was, just to make sure we are not breaking the whole ConanCenter that relies on ``.names``?

Reply:
Maybe just leaving the test as-is, and adding a new, duplicated one with ``cmake_target_name``?